node/podcontroller.go

```diff
@@ -126,17 +129,17 @@ func NewPodController(cfg PodControllerConfig) (*PodController, error) {
 	if cfg.PodInformer == nil {
 		return nil, errdefs.InvalidInput("must set informer")
 	}
-	if cfg.ConfigMapLister == nil {
+	if cfg.ConfigMapInformer == nil {
 		return nil, errdefs.InvalidInput("must set config map lister")
```

Comment:
Error definition is wrong.

Reply:
Same for all previous lister validations below.

Reply:
I don't even know if it makes sense to log it like this.

Reply:
Updated